packages/DataGrid/src/DataGrid.styles.js

```diff
@@ -0,0 +1,220 @@
+            z-index: 1;
+          }
+        `
+      : "";
+  }}
+`;
+
+export const CellHeader = styled(Cell)`
+  background: ${experimentalGrey};
+  border-bottom: 1px solid ${tokens.border.color};
+  border-left: 1px solid ${tokens.border.color};
+  color: ${tokens.color.black};
+  display: block;
+  font-weight: 600;
+  overflow: hidden;
+  padding: ${tokens.space};
+  text-overflow: ellipsis;
+  white-space: nowrap;
+  width: 100%;
+`;
+
+export const InnerCell = styled.div`
+  display: block;
+  height: 100%;
+  overflow: hidden;
+  padding: ${tokens.space};
+  text-overflow: ellipsis;
+  white-space: nowrap;
+  width: 100%;
+`;
+
+export const InnerElementType = styled.div``;
+
+export const InnerElementTypeMainGrid = styled.div``;
+
+export const OuterElementTypeMainGrid = styled.div`
+  /* The arrow navigation is sensitive with borders. If a border is used, it will cause issues keyboard navigation. */
+  box-shadow: 0px 1px 0px 0px ${tokens.border.color}, 1px 0px 0px 0px ${tokens.border.color};
+`;
+
+export const GridCell = styled.div`
+  height: 1px;
+  left: 0;
+  overflow: hidden;
+  position: absolute;
+  top: 0;
+  white-space: nowrap;
+  width: 1px;
+`;
+
+export const FillerTopRight = styled.div`
+  background: ${experimentalGrey};
+  border: 1px solid ${tokens.border.color};
+  border-bottom: 0;
+  position: absolute;
+  right: 0;
+  top: 0;
+  ${({ rowHeight, scrollBarWidth }) => {
+    const displayBlock = scrollBarWidth > 0 ? "" : "display: none;";
+
+    return `
+      ${displayBlock}
+      width: ${scrollBarWidth}px;
+      height: ${rowHeight}px;
+    `;
+  }}
+`;
+
+export const FillerBottomLeft = styled.div`
+  /* this is a small square filler on the bottom left corner of the DataGrid */
+  background: ${experimentalGrey};
+  border: 1px solid ${tokens.border.color};
+  bottom: 0;
+  left: 0;
+  position: absolute;
+  z-index: 3;
+  ${({ stickyGridWidth, scrollBarWidth }) => {
+    const displayBlock = scrollBarWidth > 0 ? "" : "display: none;";
+
+    return `
+      ${displayBlock}
+      height: ${scrollBarWidth}px;
+      width: ${stickyGridWidth}px;
+    `;
+  }}
+`;
+
+export const Flex = styled.div`
+  display: flex;
+`;
+
+export const RowCount = styled.div`
+  color: ${tokens.color.blackLighten40};
+  font-size: ${stylers.fontSize(-2)};
+`;
+
+export const WhileOnScrolling = styled.div`
+  background: ${tokens.color.cremeLighten5};
+  border-radius: ${tokens.border.radius};
+  height: 10px;
+  width: 100%;
+`;
+
+export const Blocker = styled.div`
+  align-items: center;
+  background: transparent;
+  border: 1px solid ${tokens.border.color};
+  display: flex;
+  justify-content: center;
+  left: 0;
+  position: absolute;
+  top: 0;
+  z-index: 4;
+
+  ${({ $height, $width }) => {
+    return `
+      width: ${$width}px;
+      height: ${$height}px;
+    `;
+  }}
+`;
+
+export const Footer = styled.div`
+  border: 1px solid ${tokens.border.color};
+  box-sizing: border-box;
+  font-size: 12px;
```

Comment:
Are we not using `fontSize` helper?

Reply:
Yes, but I think we don't have 12 is something like stylers.fontSize(-2) isn't? which I don't think is that descriptive. Maybe we need to review the tokens itself for the fontsize?

Reply:
actually is a custom value for now. will keep 12px.